speechbrain/dataio/audio_io.py

```diff
@@ -0,0 +1,232 @@
+"""
+Lightweight soundfile-based audio I/O compatibility layer.
+
+This module provides a minimal compatibility wrapper for audio I/O operations
+using soundfile (pysoundfile) library, replacing torchaudio's load, save, and
+info functions.
+
```

Comment:
Can we add an example here at the top of the file using tmpdir fixture that shows all three functions usage?